trp_use_case/model/use_case.py

```diff
@@ -143,6 +143,12 @@ class use_case_collection(models.Model):
     _name = 'use_case.collection'
     _description = 'Set of use cases'
 
+    x_status = fields.Selection([
+                 ('draft', 'Draft'),
+                 ('open', 'Open'),
+                 ('done', 'Done'),
+                 ], string = "Status", help = "Collection Status", default = "draft", required = True)
```

Comment:
flake8 says that equals signs for keyword arguments go without spaces (as opposed to basically anywhere else)